### PR TITLE
[macsec] Add eapol to copp config

### PR DESCRIPTION
**Pull request by byu343**
#### Why I did it
This change enables the control packets of MACsec to be processed by CPU.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

**Comment by lguohan** [comment]
@prsunny, will this break on platforms that does not support this trap id?

**Comment by prsunny** [comment]
> @prsunny, will this break on platforms that does not support this trap id?

Yes, it will break. We should have a corresponding feature name and only enable for those platforms that support it. I see that we've a [macsec ](https://github.com/Azure/sonic-buildimage/blob/master/files/build_templates/init_cfg.json.j2#L38)feature. Can the trap be named as "macsec" instead of "eapol"?

**Comment by lguohan** [comment]
@prsunny, how do you suggest to move ahead? how can we enable this trap when macsec feature is enabled?

File: files/image_config/copp/copp_cfg.j2 [hunk 1]
```diff
@@ -91,6 +91,10 @@
 		    "trap_ids": "ip2me",
 		    "trap_group": "queue1_group1"
 	    },
+	    "eapol": {
```
Comment (lguohan) [inline files/image_config/copp/copp_cfg.j2:94]: change name to eapol -> macsec, then accordingly to prince, this trap will not be installed until the feature is enabled.

Comment (lguohan) [reply files/image_config/copp/copp_cfg.j2:94]: @byu343, can you do that? then we can merge.

**Review by lguohan — APPROVED** [review]

**Pull request merged**